index.html

```diff
@@ -19,7 +19,7 @@ <h1><img src="{{ hero.image }}" alt="Envoy"></h1>
       {% endfor %}
       <div class="action-btns text-center">
         <a href="https://www.envoyproxy.io/docs/envoy/latest/start/start" class="btn btn-primary">Get Started</a>
-        <a href="https://www.envoyproxy.io/docs/envoy/latest/install/building" class="btn btn-secondary">Download</a>
+        <a href="/download" class="btn btn-secondary">Download</a>
```

Comment:
We had switched to this link because my theory was that it's more maintained / tested, and I didn't want to maintain a separate Envoy quick start in the web site.